### DISCUSSION TITLE
Google Earth Web as wallpaper

### DISCUSSION THREAD

**Discussion by Crazy-Unicooorn** [rocksdanister/lively#1285] [Q&A]
Hi! First of all, congrats on the software, it's amazing.

I am trying to have [https://earth.google.com/web/](https://earth.google.com/web/) as wallpaper but right now it doesn't seem to work. It gives me this infinite loading screen:
![image](https://user-images.githubusercontent.com/30509883/176908862-9cc21dbc-837d-4010-8bc5-237a110fede9.png)

I have tried switching the search engine from CefSharp to WebView2, but for some reason the setting resets every time:

![Lively UI WinUI_bff0lLUzfC](https://user-images.githubusercontent.com/30509883/176909356-6fa8416f-6a5b-4b00-bc58-57ece066fa02.gif)

Any help appreciated!

By the way, my "dream" is to automatically trigger the _I'm Feeling Lucky_ feature on Google Earth with every new instance. In other words, I would like to discover a new place every time my computer starts. However I can't find a direct link to trigger that feature, so I have no clue about how to do it. If someone can solve this for me, that would be amazing!

**Comment by gpsarathy** [comment]
Hi, Is there any fix workaround for this feature to enable google earth to work. I'm also thinking same above idea to change wallpaper often by "I'm Feeling lucky". But first I'm expecting at least the site to work first. Fyi, I tried the same from another [thread](https://github.com/rocksdanister/lively/discussions/834), but same result
